packages/mui-codemod/src/deprecations/all/deprecations-all.js

```diff
@@ -86,6 +89,9 @@ export default function deprecationsAll(file, api, options) {
   file.source = transformSnackbarProps(file, api, options);
   file.source = transformerTabsProps(file, api, options);
   file.source = transformerTabsClasses(file, api, options);
+  file.source = transformDrawerProps(file, api, options);
+  file.source = transformMobileStepperProps(file, api, options);
```

Comment:
Are the Drawer and Mobile stepper expected here? Shouldn't they come with their respective cherry-pick?

Reply:
Good point, the [Drawer PR](https://github.com/mui/material-ui/pull/45377) has to be merged first.